src/Content/Version.php

```diff
@@ -195,6 +202,41 @@ public function id(): VersionId
 		return $this->id;
 	}
 
+	/**
+	 * Checks if the version is the latest version
+	 */
+	public function isLatest(): bool
+	{
+		return $this->id->is(VersionId::latest());
+	}
+
+	/**
+	 * Checks if the version is locked for the current user
+	 */
+	public function isLocked(Language|string $language = 'default'): bool
+	{
+		// check if the version is locked in any language
+		if ($language === '*') {
+			foreach (Languages::ensure() as $language) {
+				if ($this->isLocked($language) === true) {
+					return true;
+				}
+			}
+
+			return false;
+		}
+
+		return $this->lock($language)->isLocked();
+	}
+
+	/**
+	 * Returns the lock object for the version
+	 */
+	public function lock(Language|string $language = 'default'): Lock
+	{
+		return Lock::for($this, Language::ensure($language));
```

Comment:
`Lock::for()` currently doesn't take a second parameter.

Reply:
☑️